examples/web-ui/material/main.smash

```diff
@@ -0,0 +1,20 @@
+// Sample functional tests of Material Design (https://material.io/)
+// Shows how to test a website that has a different navigation for desktop and mobile
+// Steps are implemented in other .smash files
+
+// Run these tests by running `smashtest --recursive` in this directory
+
+Open Chrome
+
+    Set Desktop
+    Set Mobile
+
+        - Can navigate to home page
+
+            Given I am on home page
+        
+        - Can navigate to understanding layout page
+
+            Given I am on design page
+                Given I am on layout page 
+                    Given I am on understanding layout page
```

Comment:
Only Givens? No Whens or Thens? Perfectly valid test, but the examples should look as realistic as possible.